.circleci/config.yml

```diff
@@ -1,6 +1,27 @@
-version: 2
+version: 2.1
 jobs:
-  build:
+  frontend:
+    docker:
+      - image: pennlabs/frontend-test:latest
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps9-{{ .Branch }}-{{ checksum "frontend/yarn.lock" }}
+      - run:
+          name: Install dependencies
+          command: |
+            cd frontend
+            yarn install
+      - save_cache:
+          key: deps9-{{ .Branch }}-{{ checksum "frontend/yarn.lock" }}
+          paths:
+            - "frontend/node_modules"
+      - run:
+          name: Lint
+          command: |
+            cd frontend
+            yarn lint
```

Comment:
@sigmachirality @ArmaanT may have some fancy orb stuff that you can use here to avoid code duplication, see clubs circleci config for more details.

Reply:
yeah, gonna move to react-tools tmmr at meeting

Reply:
nvm moved now lol